frontend/src/components/Modal.js

```diff
@@ -1,17 +1,21 @@
 import React from "react";
+//import { RiCloseLine } from "react-icons/ri";
 
-const Modal = () => {
+const Modal = ({ setIsOpen, responseMessage }) => {
   return (
     <>
-      <div className="darkBG"/>
+      <div className="darkBG" />
       <div className="centered">
         <div className="theModal">
           <div className="modalContent">
-            Check your Email Address for Account Verification!
+            {responseMessage}!
           </div>
           <div className="modalActions">
             <div className="actionsContainer">
-              <button className="okayBtn" onClick={() => window.location.href = "/"}>
+              <button
+                className="okayBtn"
+                onClick={() => (window.location.href = "/")}
+              >
                 Okay
               </button>
             </div>
```

Comment:
Can these two divs, modalAction and actions Container be merged into one div? className = "modalAction actionsContainer"?

Reply:
This is Noah stuff, could be something that he styled, i'll let him know.